DocsNav/scripts/insert_navbar.jl

```diff
@@ -0,0 +1,165 @@
+# insert_navbar.jl
+#
+# Usage:
+#   julia insert_navbar.jl <html-file-or-directory> <navbar-file-or-url> [--exclude "path1,path2,..."]
+#
+# Features:
+#   - Processes all .html files in the target (either a single file or recursively in a directory).
+#   - Removes any previously inserted navbar block (i.e. everything between <!-- NAVBAR START --> and <!-- NAVBAR END -->)
+#     along with any extra whitespace immediately following it.
+#   - Inserts the new navbar block immediately after the first <body> tag.
+#   - If the fetched navbar content does not contain the markers, it is wrapped (without further modification) with:
+#         <!-- NAVBAR START -->
+#         (navbar content)
+#         <!-- NAVBAR END -->
+#   - If a file does not contain a <body> tag, that file is skipped.
+#   - An optional --exclude parameter (comma‑separated) will skip any file whose path contains one of the provided substrings.
+#
+# Required package: HTTP
+#
+# (Install HTTP via Julia’s package manager, e.g. using Pkg; Pkg.add("HTTP"))
+
+using HTTP
```

Comment:
DocsNav doesn't have a install Julia step, so it won't work unless the caller manually sets up Julia before using it.

I think the easiest way out of this is to manually create a temporary environment inside this script and install the dependencies. This would be similar to what the version check script on the docs repo does: https://github.com/TuringLang/docs/blob/master/.github/workflows/version_check.jl

```suggestion

using Pkg
Pkg.activate(temp=true)
Pkg.add("HTTP")

using HTTP
```